src/Options.js

```diff
@@ -18,20 +37,10 @@ function Options(data) {
     this.style = data.style;
```

Comment:
The doc string for this function references a `.reset` function with doesn't currently exist.  It needs to be updated.